lib/pharos/kubeadm.rb

```diff
@@ -193,6 +194,24 @@ def volume_mounts_for_authentication_token_webhook
         volume_mounts
       end
 
+      def configure_audit_file(config)
+        config['apiServerExtraArgs'].merge!(
+          "audit-log-path" => @config.audit.file.path,
+          "audit-log-maxage" => @config.audit.file.max_age.to_s,
+          "audit-log-maxbackup" => @config.audit.file.max_backups.to_s,
+          "audit-log-maxsize" => @config.audit.file.max_size.to_s,
+          "audit-policy-file" => AUDIT_CFG_DIR + '/policy.yml'
```

Comment:
`policy.yml` is missing?

Reply:
nope, it's put into place at https://github.com/kontena/pharos-cluster/blob/feature/audit-logging/lib/pharos/phases/setup_master.rb#L14